Modules/MID/include/MID/RawQcCheck.h

```diff
@@ -0,0 +1,45 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   RawQcCheck.h
+/// \author My Name
+///
+
+#ifndef QC_MODULE_MID_MIDRAWQCCHECK_H
+#define QC_MODULE_MID_MIDRAWQCCHECK_H
+
+#include "QualityControl/CheckInterface.h"
+
+namespace o2::quality_control_modules::mid
+{
+
+/// \brief  Example QC Check
```

Comment:
update doxygen. Does not need to be long, just a few words to say what it is and what it does. I put this comment here but it applies to the Task as well.